From 735b8acdb3161abdf7e2c606e028eedf938043c8 Mon Sep 17 00:00:00 2001
From: Jo Shields <directhex@apebox.org>
Date: Mon, 19 Jun 2017 14:09:44 +0100
Subject: [PATCH] Explicitly use #!/bin/bash for install script (#15)

One of the line copies uses non-POSIX wildcard matching semantics, which breaks on any system where /bin/sh is not Bash or similar (e.g. most Debian derivatives)
---
 install-mono-prefix.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/install-mono-prefix.sh b/install-mono-prefix.sh
index c569cbf2..6f1b086e 100755
--- a/install-mono-prefix.sh
+++ b/install-mono-prefix.sh
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 set -e
 set -x
 if [ $# -ne 1 ]; then
-- 
2.11.0

